File: app/src/main/java/com/jds/fitnessjunkiess/getfitapp/Data/Database/WorkoutExerciseAssignmentTestData.java
package com.jds.fitnessjunkiess.getfitapp.Data.Database;

import com.jds.fitnessjunkiess.getfitapp.Data.Entities.WorkoutExerciseAssignment;
import java.util.ArrayList;
import java.util.List;

public class WorkoutExerciseAssignmentTestData {
  public static List<WorkoutExerciseAssignment> getData() {
    WorkoutExerciseAssignment workoutExerciseAssignment1 = new WorkoutExerciseAssignment();
    workoutExerciseAssignment1.setId(1);
    workoutExerciseAssignment1.setReps(12);
    workoutExerciseAssignment1.setRestTime("00:00:30");
    workoutExerciseAssignment1.setSets(3);
    workoutExerciseAssignment1.setExerciseId(1);
    workoutExerciseAssignment1.setWorkoutId(1);

    WorkoutExerciseAssignment workoutExerciseAssignment2 = new WorkoutExerciseAssignment();
    workoutExerciseAssignment2.setId(2);
    workoutExerciseAssignment2.setReps(12);
    workoutExerciseAssignment2.setRestTime("00:00:30");
    workoutExerciseAssignment2.setSets(4);
    workoutExerciseAssignment2.setExerciseId(2);
    workoutExerciseAssignment2.setWorkoutId(1);

    WorkoutExerciseAssignment workoutExerciseAssignment3 = new WorkoutExerciseAssignment();
    workoutExerciseAssignment3.setId(3);
    workoutExerciseAssignment3.setReps(12);
    workoutExerciseAssignment3.setLengthTime("00:60:00");
    workoutExerciseAssignment3.setExerciseId(5);
    workoutExerciseAssignment3.setWorkoutId(3);


    WorkoutExerciseAssignment workoutExerciseAssignment4 = new WorkoutExerciseAssignment();
    workoutExerciseAssignment4.setId(4);
    workoutExerciseAssignment4.setReps(12);
    workoutExerciseAssignment4.setRestTime("00:00:30");
    workoutExerciseAssignment4.setSets(3);
    workoutExerciseAssignment4.setExerciseId(1);
    workoutExerciseAssignment4.setWorkoutId(2);

    WorkoutExerciseAssignment workoutExerciseAssignment5 = new WorkoutExerciseAssignment();
    workoutExerciseAssignment5.setId(5);
    workoutExerciseAssignment5.setReps(12);
    workoutExerciseAssignment5.setRestTime("00:00:30");
    workoutExerciseAssignment5.setSets(4);
    workoutExerciseAssignment5.setExerciseId(2);
    workoutExerciseAssignment5.setWorkoutId(2);

    WorkoutExerciseAssignment workoutExerciseAssignment6 = new WorkoutExerciseAssignment();
    workoutExerciseAssignment6.setId(6);
    workoutExerciseAssignment6.setReps(12);
    workoutExerciseAssignment6.setRestTime("00:00:30");
    workoutExerciseAssignment6.setSets(4);
    workoutExerciseAssignment6.setExerciseId(4);
    workoutExerciseAssignment6.setWorkoutId(2);

    List<WorkoutExerciseAssignment> workoutExerciseAssignments = new ArrayList<>();
    workoutExerciseAssignments.add(workoutExerciseAssignment1);
    workoutExerciseAssignments.add(workoutExerciseAssignment2);
    workoutExerciseAssignments.add(workoutExerciseAssignment3);
    workoutExerciseAssignments.add(workoutExerciseAssignment4);
    workoutExerciseAssignments.add(workoutExerciseAssignment5);
    workoutExerciseAssignments.add(workoutExerciseAssignment6);


    return workoutExerciseAssignments;
  }
}

File: app/src/main/java/com/jds/fitnessjunkiess/getfitapp/data/database/testData/WorkoutExerciseAssignmentTestData.kt
package com.jds.fitnessjunkiess.getfitapp.data.database.testData

import com.jds.fitnessjunkiess.getfitapp.data.dataModels.ExerciseAssignment
import java.util.ArrayList

object WorkoutExerciseAssignmentTestData {
    val data: List<ExerciseAssignment>
        get() {
            val workoutExerciseAssignment1 = ExerciseAssignment()
            workoutExerciseAssignment1.id = 1
            workoutExerciseAssignment1.reps = 12
            workoutExerciseAssignment1.restTime = "00:00:30"
            workoutExerciseAssignment1.sets = 3
            workoutExerciseAssignment1.exerciseId = 1
            workoutExerciseAssignment1.workoutId = 1

            val workoutExerciseAssignment2 = ExerciseAssignment()
            workoutExerciseAssignment2.id = 2
            workoutExerciseAssignment2.reps = 12
            workoutExerciseAssignment2.restTime = "00:00:30"
            workoutExerciseAssignment2.sets = 4
            workoutExerciseAssignment2.exerciseId = 2
            workoutExerciseAssignment2.workoutId = 1

            val workoutExerciseAssignment3 = ExerciseAssignment()
            workoutExerciseAssignment3.id = 3
            workoutExerciseAssignment3.reps = 12
            workoutExerciseAssignment3.lengthTime = "00:60:00"
            workoutExerciseAssignment3.exerciseId = 5
            workoutExerciseAssignment3.workoutId = 3


            val workoutExerciseAssignment4 = ExerciseAssignment()
            workoutExerciseAssignment4.id = 4
            workoutExerciseAssignment4.reps = 12
            workoutExerciseAssignment4.restTime = "00:00:30"
            workoutExerciseAssignment4.sets = 3
            workoutExerciseAssignment4.exerciseId = 1
            workoutExerciseAssignment4.workoutId = 2

            val workoutExerciseAssignment5 = ExerciseAssignment()
            workoutExerciseAssignment5.id = 5
            workoutExerciseAssignment5.reps = 12
            workoutExerciseAssignment5.restTime = "00:00:30"
            workoutExerciseAssignment5.sets = 4
            workoutExerciseAssignment5.exerciseId = 2
            workoutExerciseAssignment5.workoutId = 2

            val workoutExerciseAssignment6 = ExerciseAssignment()
            workoutExerciseAssignment6.id = 6
            workoutExerciseAssignment6.reps = 12
            workoutExerciseAssignment6.restTime = "00:00:30"
            workoutExerciseAssignment6.sets = 4
            workoutExerciseAssignment6.exerciseId = 4
            workoutExerciseAssignment6.workoutId = 2

            val workoutExerciseAssignments = ArrayList<ExerciseAssignment>()
            workoutExerciseAssignments.add(workoutExerciseAssignment1)
            workoutExerciseAssignments.add(workoutExerciseAssignment2)
            workoutExerciseAssignments.add(workoutExerciseAssignment3)
            workoutExerciseAssignments.add(workoutExerciseAssignment4)
            workoutExerciseAssignments.add(workoutExerciseAssignment5)
            workoutExerciseAssignments.add(workoutExerciseAssignment6)


            return workoutExerciseAssignments
        }
}
